readme.txt

```diff
@@ -181,26 +181,22 @@ INTERESTED IN DEVELOPMENT?
 
 = 4.0.0 - 2020-03-03 =
 
-* Add - Add woocommerce_admin_process_variation_object hook before $variation->save(), pass WC_Product_Variation object to hooks in the variation metabox. #24929
-* Add - Added support for placeholder attribute in quantity inputs. #25418
-* Add - Added tax_status and tax_class columns to the product meta data lookup table. #25428
-* Add - Added woocommerce_top_rated_widget_args filter for WC_Widget_Top_Rated_Products widget. #25320
-* Dev - Add actions before and after grouped product list to allow adding custom rows. #25093
-* Dev - Add filter to tweak whether a product has enough stock while attempting to pay for an order. #25230
-* Dev - Added the automattic/jetpack-constants package and replace PHP constant definition checks with it. #25516
-* Dev - Adds a triggerHandler called checkout_place_order_success on a successful order during the checkout process. This allows plugin develops to hook into the JavaScript process that gets triggered on the client side on order checkout. #25289
-* Dev - Allow filtering of default meta value inside WC_Data::get_meta even if meta key not found. #24066
-* Dev - Includes Emogrifier composer package instead of including into includes/libraries. #25525
-* Dev - Introduce WC_Countries::get_vat_countries for returning a list of countries that uses VAT and refactor WC_Countries::get_european_union_countries with backward compatibility and deprecation to remove the VAT functionality from there. Brexit, remove GB from WC_Countries::get_european_union_countries. #24943
-* Dev - Introduced woocommerce_download_product_filepath filter. #25485
-* Dev - Introduced woocommerce_email_content_type filter. Updated woocommerce_email_from_name filter arguments to include default wp_email() "from" name. Updated woocommerce_email_from_address filter arguments to include default wp_email() email address. #25405
-* Dev - Introduced woocommerce_shortcode_products_query_results filter. #25573
-* Dev - Removed hash_equals() polyfill as it was no longer needed. #25474
-* Dev - Removed unused .order-actions CSS. #25581
 * Enhancement - Included information about packages in the System Status Report. #25584
```

Comment:
`Enhancement` is always the first section, so people can easily find what is new.